### PR TITLE
(feat) O3 3783 : Update styles for tooltip

### PR DESCRIPTION
**Pull request by Vijaykv5**
## Requirements
- [x] This PR has a title that briefly describes the work done including a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) type prefix and a Jira ticket number if applicable. See existing PR titles for inspiration.

#### For changes to apps
- [x]  My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide) and [**design documentation**](https://zeroheight.com/23a080e38/p/880723-introduction).

#### If applicable
- [ ] My work includes tests or is validated by existing tests.

## Summary

The PR updates the styles for the steps and placement of buttons inside tooltip.

- [x] To have a **close button** on the tooltip. _(top-right end of tooltip)_
- [x] Align the display of steps and placement of buttons inside tooltip.
- [x] Remove the arrow from the Finish button.
- [x] Make the `disableBeacon` property default for all steps. b4147b6b6893929fc32d3b7d7aca783ae31f4e24

## Screenshots

Tooltip which have no `heading` 
Screenshot 2024-08-11 at 5 31 22 PM

Tooltip which have `heading`
Screenshot 2024-08-11 at 5 28 24 PM

Tooptip for last step ( finish button )
Screenshot 2024-08-11 at 5 28 54 PM

## Related Issue

[O3-3783](https://openmrs.atlassian.net/browse/O3-3783)

## Other

Clicking over the `close` icon skips the entire tutorial ( not that particular step )

[O3-3783]: https://openmrs.atlassian.net/browse/O3-3783?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/root.component.tsx [hunk 1]
```diff
@@ -10,11 +10,19 @@ const RootComponent: React.FC = () => {
...
+      disableBeacon: true,
+    }));
+  };
+
   useDefineAppContext<TutorialContext>('tutorial-context', {
     showTutorial,
     steps,
     setShowTutorial: (showTutorial: boolean) => setShowTutorial(showTutorial),
-    setSteps: (steps: Step[]) => setSteps(steps)
+    setSteps: (steps: Step[]) => setSteps(updateStepsWithDisableBeacon(steps)),
```
Comment (jayasanka-sack) [inline src/root.component.tsx:25]:
```suggestion
    setSteps: updateSteps,
```

File: src/root.component.tsx [hunk 2]
```diff
@@ -10,11 +10,21 @@ const RootComponent: React.FC = () => {
   const [steps, setSteps] = React.useState<Step[]>([]);
   const [stepIndex, setStepIndex] = React.useState(0);
 
+  // To disable the beacon for all steps
```
Comment (jayasanka-sack) [inline src/root.component.tsx:13]:
```suggestion
  // Set steps with default step options
```

**Comment by jayasanka-sack** [comment]
@Vijaykv5 Can I some latest screenshots please?

**Comment by Vijaykv5** [comment]
**Latest Changes :** 
Screenshot 2024-08-17 at 8 47 49 PM



<details><summary>Suggested UI change</summary>

<p>
This is as per the design we have discussed over our weekly call.  </p>

Screenshot 2024-08-17 at 8 51 32 PM

</p>
</details>

**Comment by jayasanka-sack** [comment]
I update the stylings of the close button.
Screenshot 2024-08-19 at 14 50 56

**Comment by jayasanka-sack** [comment]
Thanks @Vijaykv5 !

**Pull request merged**